pkg/probe/http/http_test.go

```diff
@@ -158,13 +158,21 @@ func TestHTTPProbeChecker(t *testing.T) {
 			handler:    headerCounterHandler,
 			reqHeaders: http.Header{},
 			health:     probe.Success,
-			accBody:    "4",
+			accBody:    "3",
 		},
 		{
 			handler:    headerKeysNamesHandler,
 			reqHeaders: http.Header{},
 			health:     probe.Success,
-			accBody:    "Accept\nAccept-Encoding\nConnection\nUser-Agent",
+			accBody:    "Accept\nConnection\nUser-Agent",
+		},
+		{
+			handler: headerEchoHandler,
+			reqHeaders: http.Header{
+				"Accept-Encoding": {"gzip"},
```

Comment:
also please add a test for the non-strandard encoding value

Reply:
👍 done, ty